Update issue templates (#184)
twilio/twilio-voice-react-native: .github/ISSUE_TEMPLATE/bug_report.md

> 

Reviewer: Can we add a title here?

```
# Submission Checklist
```

> # Submission Checklist

Reviewer: ```suggestion
# Issue

## Pre-submission Checklist
```

- [ ] I have verified that the issue occurs with the latest release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-voice-react-native/blob/main/CHANGELOG.md).
- [ ] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice-react-native/blob/main/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
- [ ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

> **Describe the bug**

Reviewer: ```suggestion
## Description of Issue
```

> **Describe the Issue**

Reviewer: ```suggestion
## Description
```

A clear and concise description of what the bug is.

> **To Reproduce**

Reviewer: ```suggestion
## Reproduction Steps
```

> Steps to reproduce the behavior:

Reviewer: ```suggestion
```

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

> **Expected behavior**

Reviewer: ```suggestion
## Expected Behavior
```

## Actual Behavior

> 

Reviewer: ```suggestion
```

> **Actual Behavior**

Reviewer: ```suggestion
## Actual Behavior
```

What actually happens.

## Reproduction Frequency

> 

Reviewer: ```suggestion
```

> **Reproduces How Often**

Reviewer: ```suggestion
## Reproduction Frequency
```

> What percentage of the time does it reproduce?

Reviewer: ```suggestion
Is the reproducibility of the issue deterministic? If not, what percentage of the time does the issue occur? In how many attempts was the issue observed?
```

> **Screenshots**

Reviewer: ```suggestion
## Screenshots
```

If applicable, add screenshots to help explain your problem.

> **Software and Device Information (please complete the following information):**

Reviewer: ```suggestion
## Software and Device Information
**Please complete the following information.**
```

 - Device: [e.g. iPhone 12, Pixel 3]
 - OS: [e.g. iOS 16, Android API Level 31]

>  - React Version

Reviewer: ```suggestion
 - React version: [e.g. 18.1.0]
```

>  - React Native Version

Reviewer: ```suggestion
 - React Native version: [e.g. 0.70.9]
```

>  - Node Version

Reviewer: ```suggestion
 - Node version: [e.g. 16.18.1]
```

>  - npm or yarn version

Reviewer: ```suggestion
 - npm or yarn version: [e.g. 8.19.2, 1.22.19]
```

> **Additional context**

Reviewer: ```suggestion
## Additional Context
```

---
name: Bug report
about: Create a report to help us improve
title: ''
labels: ''
assignees: ''

---

> > Please ensure that you are not sharing any
> [Personally Identifiable Information(PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
> or sensitive account information (API keys, credentials, etc.) when reporting an issue.

Reviewer 2: ```suggestion
- [ ] I have verified that the issue occurs with the latest release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-voice-react-native/blob/main/CHANGELOG.md).
- [ ] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice-react-native/blob/main/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
- [ ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.
```

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Actual Behavior**

What actually happens.

**Reproduces How Often**

What percentage of the time does it reproduce?

**Screenshots**
If applicable, add screenshots to help explain your problem.

> **Smartphone (please complete the following information):**

Reviewer 2: ```suggestion
**Software and Device Information (please complete the following information):**
```

 - Device: [e.g. iPhone, Pixel3]
 - OS: [e.g. iOS16, Android API Leveln 31]

>  - Version [e.g. 22]

Reviewer 2: What is "Version" in this context? The SDK version?

Reviewer: +1, let's ask for as much info as possible. React version, react native version, node version, npm or yarn version

- [ ] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice-react-native/blob/main/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
- [ ] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Actual Behavior**

What actually happens.

**Reproduces How Often**

What percentage of the time does it reproduce?

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Software and Device Information (please complete the following information):**

>  - Device: [e.g. iPhone, Pixel3]

Reviewer: ```suggestion
 - Device: [e.g. iPhone, Pixel 3]
```

>  - OS: [e.g. iOS16, Android API Leveln 31]

Reviewer: ```suggestion
 - OS: [e.g. iOS16, Android API Level 31]
```
